Cargo.toml

```diff
@@ -204,7 +382,9 @@ move-core-types = { path = "external-crates/move/move-core/types", features = ["
 move-disassembler = { path = "external-crates/move/tools/move-disassembler" }
 move-package = { path = "external-crates/move/tools/move-package" }
 move-unit-test = { path = "external-crates/move/tools/move-unit-test" }
+move-stdlib = { path = "external-crates/move/move-stdlib" }
```

Comment:
ditto, execution-layer.

Reply:
See my other comment,, listing these as "workspace dependencies" doesn't implicitly make things depend on these crates, it just makes the usage of them a bit easier elsewhere in the codebase. Listing these here does not impact any of the work you want to do with the execution layer, id argue it makes it easier